opm/grid/polyhedralgrid/geometry.hh

```diff
@@ -41,6 +45,70 @@ namespace Dune
     typedef Dune::FieldVector< ctype, coorddimension > GlobalCoordinate;
     typedef Dune::FieldVector< ctype, mydimension >    LocalCoordinate;
 
+    typedef typename Grid::Traits::ExtraData  ExtraData;
+    typedef typename Grid::Traits::template Codim<codimension>::EntitySeed EntitySeed;
+
```

Comment:
@atgeirr : You can start reading from here.

Reply:
Thanks, I looked at it and saw some things that were new to me. I did notice a minor possible bug, will make a note in the diff.